cocos/2d/CCScene.cpp

```diff
@@ -167,7 +172,7 @@ void Scene::render(Renderer* renderer)
         //clear background with max depth
         camera->clearBackground(1.0);
         //visit the scene
-        visit(renderer, transform, 0);
+        visit(renderer, transform, Node::FLAGS_TRANSFORM_DIRTY);
```

Comment:
@ricardoquesada Hi, Riq, there are some bugs when visiting the scene by multiple camera, because of the cached matrix does not work correctly. Passing the dirty flag in this function will make the cache invalid, but with performance degeneration. This bugs is missed, so let us record it, find the real bug and fix it, then we can remove this flag.